code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -69,6 +68,8 @@
 	if(!istype(location) || !(location.air))
 		return 0
 
+	location.active_hotspot = src
```

Comment:
Necessary because hotspot_expose() can create a hotspot and immediately call fire_act() on objects on the tile, which in the case of light fixtures causes sparks (because they take damage), and those sparks call hotspot_expose(), and before you know it you have 10 hotspot on the same turf.